kubetest/extract_k8s.go

```diff
@@ -373,6 +373,20 @@ func (e extractStrategy) Extract(project, zone string, extractSrc bool) error {
 		if zone == "" {
 			return fmt.Errorf("--gcp-zone unset")
 		}
+		if strings.HasSuffix(e.option, "-latest") {
+			// get latest supported master version
+			res, err := output(exec.Command("gcloud", "container", "get-server-config", fmt.Sprintf("--project=%v", project), fmt.Sprintf("--zone=%v", zone), "--format=value(validMasterVersions)"))
```

Comment:
Which environment is this command running against?

gke-staging-latest = the latest version available of GKE staging. So, we should make sure two things:
1. This command is running against gke-staging environment.
2. get-server-config on the GKE staging environment returns the latest version on the staging environment (even if that version hasn't rolled out to prod).

Reply:
the endpoint is already handled by https://github.com/krzyzacy/test-infra/blob/6168a28bc1952da93339c45c6718473d2978958f/kubetest/gke.go#L168-L183

Reply:
Cool. Let's verify (2). LGTM otherwise.

Reply:
@foxish can you confirm (2) is correct in this case?